src/main/java/com/example/demo/service/EmployeeService.java

```diff
@@ -54,12 +59,14 @@ public Employee deleteEmployee(Long id) {
     private Employee convertEntityToEmployee(EmployeeEntity employeeEntity) {
         Employee employee = new Employee();
         employee.setName(employeeEntity.getName());
-        if(employeeEntity.getId() != null) {
+        if (employeeEntity.getId() != null) {
             employee.setId(employeeEntity.getId());
         }
         employee.setJob(employeeEntity.getJob());
         employee.setDepartmentId(employeeEntity.getDepartmentEntity().getId());
         employee.setDepartmentName(employeeEntity.getDepartmentEntity().getName());
+        if (employeeEntity.getTeamEntity() != null)
+            employee.setTeamId(employeeEntity.getTeamEntity().getId());
```

Comment:
I would like to see the team name, not the id

Reply:
resolved